experimental/dds/tree/src/SharedTree.ts

```diff
@@ -408,6 +408,10 @@ export class SharedTree extends SharedObject {
 		this.editLog.addLocalEdit(edit);
 		this.emit(SharedTreeEvent.EditCommitted, edit.id);
 	}
+
+	protected rebaseOpCore() {
```

Comment:
In the meeting a week ago we have chatted that "rebase" is not the right term to use here, as we are not actually rebasing anything (and it confused people).
I think we need to find better name for it that reflects what it does - it applies local change that is saved as local op.

Reply:
Personally I don't see a problem with "rebase." I think "replay" was suggested in the meeting, did you have any other ideas?

Reply:
replayLocalOp maybe better?
It's not rebase because ops are applied to exactly same base, so it's more like fetch or push (in git terminology) - commits are applied as is. Actually, it's more like stash :) So maybe we can use this analogy here - applyStash and getStash? I think some people will get confused by that.